src/Dfc.CourseDirectory.WebV2/Features/Venues/AddVenue/Published.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@model Dfc.CourseDirectory.WebV2.Features.Venues.AddVenue.Published.ViewModel
+@{
+    Layout = "_V2LayoutProviderContext";
+    ViewContext.SetLayoutData(title: "Location added");
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <govuk-panel heading-level="1" class="govuk-!-margin-bottom-9">
+            <govuk-panel-title>
+                Location added
+            </govuk-panel-title>
+            <span data-testid="VenueName">@Model.Name</span>
+        </govuk-panel>
+
+        <h2 class="govuk-heading-m">What next?</h2>
+        <ul class="govuk-list">
+            <li>
+                <a asp-action="Index" asp-all-route-data="@ProviderContext.RouteValues" class="govuk-link">
```

Comment:
I'd rather see an explicit `asp-controller` params in views for safety (no 1-1 mapping between a view and a controller, so no guarantee that the correct action rendered this view)

Reply:
Yep - that was missed.